contracts/staking/Staking.sol

```diff
@@ -717,6 +717,14 @@ contract Staking is StakingV1Storage, GraphUpgradeable, IStaking {
         _withdraw(msg.sender);
     }
 
+    /**
+     * @dev Set the destination where to send rewards.
+     * @param _destination Rewards destination address. If set to zero, rewards will be restaked
+     */
+    function setRewardsDestination(address _destination) public override {
+        rewardsDestination[msg.sender] = _destination;
```

Comment:
Ariel slacked me saying he will add an event here, good idea

Reply:
Can we make it external? This is what the interface shows, seems we are no using it within the contract

Reply:
event added :)

Reply:
I'll change it to be external. An event was added to this function too.

Reply:
Will _destination be set up via Remix or some key in Agent config?

Reply:
`setRewardsDestination()` is a function that needs to be called by the indexer address (the one that staked), in the case of an indexer through a TokenLock it needs be called initially through Remix. The feature can be potentially integrated in the frontend too.